src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -197,6 +197,9 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     ov::pass::Manager manager;
     manager.set_per_pass_validation(false);
     CPU_REGISTER_PASS_COMMON(manager, ov::pass::InitNodeInfo);
+    CPU_REGISTER_PASS_COMMON(manager, ov::pass::MarkShapeOfSubgraphs);
+    // todo: should be uncommented when xxx-105060 is ready
+     CPU_REGISTER_PASS_COMMON(manager, ov::pass::KeepConstAndDecompression);
```

Comment:
What should be uncommented?

Reply:
Registering `KeepConstAndDecompression` should be uncommented when CPU will be ready to deal with unfolded graphs. 
On this commit i left if uncommented by mistake. We agreed to merge mechanism in this PR without activating it yet